src/controllers/activegate/reconciler/statefulset/container_statsd.go

```diff
@@ -0,0 +1,105 @@
+package statefulset
+
+import (
+	"fmt"
+
+	"github.com/Dynatrace/dynatrace-operator/src/controllers/activegate/internal/consts"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const statsDProbesPortName = "statsd-probes"
+const statsDProbesPort = 14999
+
+var _ ContainerBuilder = (*StatsD)(nil)
+
+type StatsD struct {
+	GenericContainer
+}
+
+func NewStatsD(stsProperties *statefulSetProperties) *StatsD {
+	return &StatsD{
+		GenericContainer: *NewGenericContainer(stsProperties),
+	}
+}
+
+func (statsd *StatsD) BuildContainer() corev1.Container {
+	return corev1.Container{
+		Name:            consts.StatsDContainerName,
+		Image:           statsd.StsProperties.DynaKube.ActiveGateImage(),
+		ImagePullPolicy: corev1.PullAlways,
+		Env:             statsd.buildEnvs(),
+		VolumeMounts:    statsd.buildVolumeMounts(),
+		Command:         statsd.buildCommand(),
+		Ports:           statsd.buildPorts(),
+		ReadinessProbe: &corev1.Probe{
+			ProbeHandler: corev1.ProbeHandler{
+				HTTPGet: &corev1.HTTPGetAction{
+					Path: "/readyz",
+					Port: intstr.IntOrString{IntVal: statsDProbesPort},
+				},
+			},
+			InitialDelaySeconds: 10,
+			PeriodSeconds:       15,
+			FailureThreshold:    3,
+			SuccessThreshold:    1,
+			TimeoutSeconds:      1,
+		},
+		LivenessProbe: &corev1.Probe{
+			ProbeHandler: corev1.ProbeHandler{
+				HTTPGet: &corev1.HTTPGetAction{
+					Path: "/livez",
```

Comment:
Actually, `/healthz` [is deprecated and not recommended in Kubernetes since 1.16](https://kubernetes.io/docs/reference/using-api/health-checks/#api-endpoints-for-health) so I went with the `/livez`, `/readyz` convention for EEC & StatsD instead.